contracts/Certificate.sol

```diff
@@ -286,6 +286,18 @@ contract Certificate is
     return _purchaseAmounts[certificateId];
   }
 
+  /**
+   * @notice This function is unsupported and will always revert.
+   * @dev Override to disable ERC721 operator approvals, since certificate tokens are non-transferable.
+   */
+  function approve(address, uint256)
+    public
+    payable
+    override(ERC721AUpgradeable, IERC721AUpgradeable)
+  {
+    revert FunctionDisabled();
+  }
+
```

Comment:
is this related to the upgrade, or something that was an oversight by us originally?

Reply:
I think there function signature changed from pure to payable in the latest version which in also necessitated a reordering for solhint

Reply:
oh I see - I actually hadn't noticed the deleted version of this same function, that totally makes sense